src/python-common/ceph/deployment/service_spec.py

```diff
@@ -126,12 +126,16 @@ def __init__(self, label=None, hosts=None, count=None, all_hosts=False, host_pat
         #: An fnmatch pattern to select hosts. Can also be a single host.
         self.host_pattern = host_pattern
 
+        # flag to disable any automated placement
+        self.unmanaged = unmanaged
```

Comment:
should this live in `PlacementSpec` or `ServiceSpec`?

Reply:
hmm.  for the code perspective, that probably makes more sense.  but... the are parsing the arg with PlacementSpec.from_string('-').  So we either special-case all of those places so that - means 'unmanaged spec' and other strings mean 'parse this placement'... or we keep it in placementspec ("placement is not managed").

Reply:
or... we make an ``--unmanaged`` flag for each 'orch apply' CLI and keep it separate from placement.

Reply:
`ceph orch prometheus apply -` looks like it wants to read the spec form stdin. Wondering, if `-` is actually the right way to set the `unmanaged` flag.

Reply:
:+1: for `--unmanaged`